spacemacs/packages.el

```diff
@@ -136,6 +137,35 @@ which require an initialization must be listed explicitly in the list.")
       (setq ace-jump-mode-scope 'global)
       (evil-leader/set-key "`" 'ace-jump-mode-pop-mark))))
 
+(defun spacemacs/init-ace-link ()
+  (use-package ace-link
+    :defer t
+    :init (ace-link-setup-default)
```

Comment:
I don't get this. What if a user who doesn't use `spacemacs/ace-buffer-liks` but still want `ace-link` feature in `help mode` buffer, for example? I need a way to trigger `ace-link-setup-default` function https://github.com/abo-abo/ace-link/blob/master/ace-link.el#L217-L226 to bind `o` in `help-mode-map`